CodeGeneration/Sources/generate-swift-syntax/ChildNodeChoices.swift

```diff
@@ -15,33 +15,12 @@ import SwiftSyntaxBuilder
 import SyntaxSupport
 
 struct ChildNodeChoices {
-  struct Choice: IdentifierConvertible, TypeConvertible {
-    var documentation: SwiftSyntax.Trivia
+  @dynamicMemberLookup
+  struct Choice {
```

Comment:
I resorted to this ugly hack because for some reason the code wouldn't compile when I used `any NodeChoiceConvertible`, maybe I'd done something wrong?